move2kube/m2k.sw.yml

```diff
@@ -96,6 +96,8 @@ states:
             message: '"Please go to your <a href=\"" + $SECRET.move2kube_url + "/workspaces/"+ .workspaceId + "/projects/" + .projectId + "/outputs/"+ .transformId + "\">Move2Kube instance</a> and answers to questions in order to continue the Move2Kube workflow"'
             origin: "Move2Kube Workflow"
             topic: "Move2Kube Workflow"
+            targetUsers:
```

Comment:
I don't think it's needed, is it? I was removed from notifications API

Reply:
It depends on what is wanted:
with targetUsers the message goes to personal notification. without it it goes to updates.
in this context, it seems like a personal message, so having it here.

Reply:
I mean: targetUsers disappeared from notifications API, you cannot use it

Reply:
Removed from the latest commit

Reply:
so their API is outdated?